src/screens/about/index.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+class About extends React.Component {
+  render() {
+    return (
+      <div>
+        {/* the issues and source links will live in the header*/}
+        <a href="https://github.com/FormidableLabs/component-playground">Source Code on GitHub</a>
+        <br/>
+        <a href="https://github.com/FormidableLabs/component-playground/issues">Report an Issue</a>
```

Comment:
Will all OSS pages have a sidebar containing Table of Contents/Links, like Victory does? If yes, then I think these links are redundant. Otherwise... is good.